xarray/core/variable.py

```diff
@@ -1212,34 +1212,79 @@ def __setitem__(self, key, value):
         raise TypeError('%s values cannot be modified' % type(self).__name__)
 
     @classmethod
+    def _concat_numpy(cls, variables, positions=None):
```

Comment:
I'm not sure this has enough to need to be a separate function, but if so:
1. It shouldn't be a `classmethod` -- we don't need to overwrite it.
2. Docstring should follow the [numpydoc](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) style, which we use throughout the project.